docs/3.0.0-beta.x/installation/digitalocean-one-click.md

```diff
@@ -151,19 +151,10 @@ pm2 stop strapi-develop
 psql -c "ALTER USER strapi with password 'your-new-password';"
 ```
 
-- Update the `/srv/strapi/strapi/config/environments/development/database.json` file with the new password.
-
-```json
-...
-"settings": {
-  "client": "postgres",
-  "host": "127.0.0.1",
-  "port": "5432",
-  "database": "strapi",
-  "username": "strapi",
-  "password": "your-new-password"
-},
-...
+- Update the `/srv/strapi/strapi/config/.env` file with the new password.
+
+```
+DATABASE_PASSWORD=your-new-password
```

Comment:
think there is a few missing variables here lol

Reply:
The other one are supposed to be in your database.js file statically. I just udpated the current doc but it needs some changes to be more env firendly. BTW does DO support seeting env vars in there manager interface ? or is it only a VPS instance ?

Reply:
This is only a VPS instance, I wish we could set stuff in the management GUI would be nice, but unlikely